### PR TITLE
docs: update all-contributors badge to new version

### PR DESCRIPTION
**Pull request by tkoyama010**
ref https://allcontributors.org/docs/en/bot/faq

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # <img src="https://www.pyopensci.org/images/logo.png" width=100 /> pyOpenSci scientific Python Packaging Guide
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-75-orange.svg?style=flat-square)](#contributors-)
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
+[![All Contributors](https://img.shields.io/github/all-contributors/pyOpenSci/python-package-guide?color=ee8449)](#contributors)
```
Comment (tkoyama010) [inline README.md:2]:
```suggestion
[![All Contributors](https://img.shields.io/github/all-contributors/pyOpenSci/python-package-guide?color=ee8449)](#contributors-)
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,5 @@
 # <img src="https://www.pyopensci.org/images/logo.png" width=100 /> pyOpenSci scientific Python Packaging Guide
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-75-orange.svg?style=flat-square)](#contributors-)
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
+[![All Contributors](https://img.shields.io/github/all-contributors/pyOpenSci/python-package-guide?color=ee8449)](#contributors-)
```
Comment (lwasser) [inline README.md:2]: hey @tkoyama010 i'm not sure if all-contributors bot will be able to update this without the html comments. it's an automated process. can you help us understand what the goal of this pr is? many thanks!!

Comment (lwasser) [reply README.md:2]: oh gosh sorry i just saw your link. so this is the new way of using the badge that all-contributors will recognize? is that correct? my apologies if i didn't understand!

**Review by lwasser — APPROVED** [review]
We can merge this once ci is complete

**Comment by lwasser** [comment]
ok this is failing ci because of an unrelated url issue. we can merge.

**Review by lwasser — APPROVED** [review]

**Pull request merged**